.github/workflows/release.yml

```
@@ -186,7 +186,7 @@ jobs:
       - name: Build ARM
         continue-on-error: true
         if: ${{ matrix.image.platform == 'linux/arm64/v8' }}
-        run: docker run --platform ${{ matrix.image.platform }} -v $PWD:/src -w /src/beaver jackalcooper/beaver-livebook-${{ matrix.image.suffix }}:latest bash scripts/build-for-publish.sh
+        run: docker run --platform ${{ matrix.image.platform }} -v $PWD:/src -w /src/beaver ${{ vars.DOCKERHUB_USERNAME }}/beaver-livebook-${{ matrix.image.suffix }}:latest bash scripts/build-for-publish.sh
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:55: Double quote to prevent globbing and word splitting

(shellcheck)